roscpp_azure_iothub/src/ros_azure_iothub_cpp_node.cpp

```diff
@@ -333,38 +453,28 @@ static int InitializeAzureIoTHub(ROS_Azure_IoT_Hub* iotHub)
     (void)IoTHubDeviceClient_SetConnectionStatusCallback(iotHub->deviceHandle, connection_status_callback, NULL);
     (void)IoTHubDeviceClient_SetDeviceTwinCallback(iotHub->deviceHandle, deviceTwinCallback, iotHub);  //Device Twin callback requires context to send message and device Handle
 
-    return 0;
+    return true;
 }
 
+static void DeinitializeAzureIoTHub(ROS_Azure_IoT_Hub* iotHub)
+{
+    ROS_INFO("Deinitializing IoTHub Device client\r\n");
```

Comment:
nit: I think it is okay to remove `\r\n` from ROS logging API.

Reply:
Sure. Code updated.